app/Http/Controllers/HomeController.php

```diff
@@ -28,19 +28,19 @@ public function __construct()
      * @return \Illuminate\Contracts\Support\Renderable
      */
     public function index() {
-        
+
         return view('welcome');
     }
 
     public function dashboard()
     {
         $user = Auth::user();
-        
+
         $userTournaments = null;
         $tournamentsWhereUserIsNotParticipant = null;
 
         if (Auth::check()) {
-            $participantRoleId = Role::all()->firstWhere('name', '=', 'participant')->id;
+            $participantRoleId = Role::where('name', RoleEnum::PARTICIPANT )->first()->id;
```

Comment:
Waarom word er geen ENUM gebruikt voor ORGANIZER maar wel voor PARTICIPANT? Wat is het verschil??